paddle/fluid/primitive/rule/vjp/details.h

```diff
@@ -2251,6 +2251,31 @@ void swiglu_grad(const Tensor& x,
   set_output<T>(x_grad, dx);
 }
 
+template <typename T>
+void softsign_grad(const Tensor& x, const Tensor& out_grad, Tensor* x_grad) {
+  if (!x_grad) return;
+
+  Tensor x_tmp = x;
+  Tensor out_grad_tmp = out_grad;
+
+  bool need_cast = is_half_dtype(x.dtype());
+  if (need_cast) {
```

Comment:
no need

Reply:
fix in commit [3ea2ae2](https://github.com/PaddlePaddle/Paddle/pull/66450/commits/3ea2ae2e2294d3870de05f88e148709f8f5ba147)